device-posture/device-posture-clear.https.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script>
+'use strict';
+
+promise_test(async (t) => {
+  const originalPosture = navigator.devicePosture.type;
+  const posture = originalPosture ? 'folded' : 'continuous';
+
+  await test_driver.set_device_posture(posture);
+  assert_equals(navigator.devicePosture.type, posture);
```

Comment:
Wait for the "change" event before checking `type`.

Reply:
Fixed in f09d788860